docs/tutorials/dp-for-cross-silo-fl.md

```diff
@@ -0,0 +1,121 @@
+# Differential privacy for cross-silo federated learning
+
+## Background
+Differential privacy (DP) is a popular and powerful technique that could add another layer of privacy protection for sensitive private data, with rigorous mathematical gurantee. DP technique can be applied to both Federated Learning (FL) applications as well as non-FL applications. There are different DP algorithms available in the literature, and among them DP-SGD algorithm is a well-known one that achieves State-of-The-Art performance in terms of model utility-privacy tradeoff. This tutorial focuses on the scenario about using DP-SGD for cross-silo FL.
```

Comment:
nit: maybe `o` caps `State-of-The-Art`

Reply:
nit: is this transferable to vertical fl? if not maybe we can specify that this targets horizontal only?

Reply:
This dp tutorial focus only on cross-silo horizontal fl. I'd update accordingly.